packages/blade/src/components/Indicator/Indicator.tsx

```diff
@@ -65,7 +65,7 @@ const Indicator = ({
   const { theme } = useTheme();
 
   const fillColor = theme.colors.feedback.background[intent].highContrast;
-  const strokeColor = theme.colors.brand.gray[200];
+  const strokeColor = theme.colors.brand.gray[200].lowContrast;
```

Comment:
Ohh... was this the TS error? 

I wonder why ts couldn’t resolve this before.

Reply:
Yes, this was the issue